scripts/install/install-image-existing

```diff
@@ -269,6 +269,9 @@ if [ -e "$DEF_GRUB" ]; then
   sed -i "s/menuentry \"VyOS.*(/menuentry \"VyOS $NEWNAME (/" $def_grub_vers
   sed -i "s/menuentry \"Lost password change.*(/menuentry \"Lost password change $NEWNAME (/" $def_grub_vers
   sed -i "sX/boot/[A-Za-z0-9\.\-]*X/boot/${NEWNAME}Xg" $def_grub_vers
+  if [ -f /etc/modprobe.d/vyos_disable_ipv6.conf ]; then
+    sed -i "/^[[:space:]]*linux / {/ipv6.disable=/! s/$/ ipv6.disable=1/}" $def_grub_vers
```

Comment:
If existing parameter is ipv6.disable=0 this will leave it as-is, it should prabably behave the same as in system-ipv6.py

Reply:
this modify the "template" from default-union-grub-entry. hopefully user doesnt modify this file. so only vyos devs have to take care, dont break things. actually, only this needs:

    "/^[[:space:]]*linux / s/$/ ipv6.disable=1/"

Reply:
Ah, I see, this is OK then.